### ISSUE TITLE
Question about copyright?

### ISSUE THREAD

**Issue by dota17** [AsyncHttpClient/async-http-client#1743]
Hi, I found[ this commit ](https://github.com/AsyncHttpClient/async-http-client/commit/a12093438aa28e559c30cbbf1c7162f7d468fa2b) changing com.ning to org.asynchttpclient and some other commits updating header about copyright.
But there are some files that their copyright still are Ning Inc.

Who holds the copyright of this project or these code?Ning Inc or AsyncHttpClient Project?
And why is copyright changed? Could you tell the reason or the history?

**Comment by TomGranot** [comment]
@dota17 Great question - I am a new maintainer of the repo, and this is obviously an historical question. The honest answer is I don't know - will check with @slandelle who has been a maintainer for a long while and get back to you. Please ping if I forget!

**Comment by slandelle** [comment]
AHC's creator is @jfarcan.
He created the project when he was working for Ning, hence the Ning copyright.
Then, he moved to Sonatype where he continued working on AHC, so there are maybe some "Sonatype" copyright around.
Finally, he moved to somewhere else, and I joined, and most of the new code has a "AsyncHttpClient Project" copyright.

When some old code was replaced, I changed the copyright. But I left the original one when the original code is still mostly there, which is what usually happens for the API.

I'm no legal person, but I think it doesn't matter. What matters is the Apache 2 license.
